lib/util.js

```diff
@@ -369,6 +369,31 @@ exports.getVideoDescription = (html) => {
     new Entities().decode(exports.stripHTML(description[1])) : '';
 };
 
+/**
+ * Get video media (extra information) from html
+ * 
+ * @param {String} body
+ * @return {Object}
+ */
+exports.getVideoMedia = (body) => {
+  let mediainfo = exports.between(body,
+    '<div id="watch-description-extras">',
+    '<div id="watch-discussion" class="branded-page-box yt-card">');
+  if (mediainfo === '') {
+    return {};
+  }
+  mediainfo = new Entities().decode(mediainfo);
```

Comment:
This should be done after all the regexp searching and matching, for `match[2]`, and maybe for `match[1]`.

Otherwise, the content of a tag could be something like

    <li>Faded (Lost Stories Remix)&lt;/li&gt; this part is ignored although it shouldn't be</li>